src/app/world-association-of-home-army-soldiers/memorials/page.tsx

```diff
@@ -1,7 +1,20 @@
-export default function MemorialsPage() {
+import { PlaceItem } from "@/components/PlaceItem/PlaceItem";
+import { memorialPlaces } from "@/data/memorialPlacesData";
+
+const MemorialPage: React.FC = () => {
```

Comment:
No need for using `React.FC`. Let TypeScript infer the returned type.
Type only what is essential and let TS do the hard work 😉 

Also I think it would be good to have a consistent convention across the project about Next exports and other exports such as components etc.

- I'd suggest to use default exported functions for Next Pages and API Routes/Handlers etc.
```tsx
export default function PageName() {}
``` 
- And named exports for any other things.
```tsx
export const SomeComponent = () => <div></div>
```